crates/biome_css_parser/src/syntax/mod.rs

```diff
@@ -370,6 +473,23 @@ pub(crate) fn parse_regular_identifier(p: &mut CssParser) -> ParsedSyntax {
     parse_identifier(p, CssLexContext::Regular)
 }
 
+pub(crate) fn is_at_url_value_raw(p: &mut CssParser) -> bool {
```

Comment:
I believe that it's better name for the function is `is_at_url_value` because we check here both possible variants.
```
AnyCssUrlValue =
	CssUrlValueRaw
	| CssString
```